recipe/meta.yaml

```diff
@@ -43,8 +43,9 @@ requirements:
     - setuptools
```

Comment:
```suggestion
```
`setuptools` has been removed in v2.12.0, see https://github.com/scrapy/scrapy/blob/2.12.0/docs/news.rst#modified-requirements